src/Database/FunctionsBuilder.php

```diff
@@ -31,11 +31,12 @@ class FunctionsBuilder
      * @param string $name the name of the SQL function to constructed
      * @param array $params list of params to be passed to the function
      * @param array $types list of types for each function param
+     * @param string $return The return type of the function expression
      * @return FunctionExpression
      */
-    protected function _build($name, $params = [], $types = [])
+    protected function _build($name, $params = [], $types = [], $return = 'string')
```

Comment:
Can we make this steing a class constant? Better to use then from the outside and cleaner as magic strings then IMO.